ceno_emul/src/rv32im.rs

```diff
@@ -182,335 +173,96 @@ pub enum InsnKind {
     SB,
     SH,
     SW,
-    /// ECALL and EBREAK etc.
-    EANY,
```

Comment:
`EANY` was never a real instruction.  It was only ever there to work around Risc0's _interesting_ ideas about decoding.